lib/contentful_lite/version.rb

```diff
@@ -1,3 +1,3 @@
 module ContentfulLite
-  VERSION = "1.0.2".freeze
+  VERSION = "1.0.3".freeze
```

Comment:
@HQ063 I believe this should at least be a minor version bump for semvar `1.1.0` due to the dependencies changing so drastically this isn't a patch.